### ISSUE TITLE
Causes HUGE spikes in power usage

### ISSUE THREAD

**Issue by electr0sheep** [rwatts3/atom-ide-vue#10]
![kapture 2018-06-04 at 13 55 38](https://user-images.githubusercontent.com/7546899/40938443-231230bc-67ff-11e8-9080-59d498314b74.gif)

**Comment by romans127** [comment]
Definitely not cool ! 
Looking into this now, thanks for reporting.
Please also post any suggestions you think may be causing this. 
I'll debug and see what I can find

**Comment by neversun** [comment]
Version 0.1.8 (`apm install ide-vue@0.1.8`) does not have this issue.

I saw some changes to vue-language-server. Maybe there is an issue on their side? Without looking into it, they did some changes to the file watcher after 0.0.31.

**Comment by romans127** [comment]
Yes , I made some minor changes, nothing that should have caused the spike in CPU. 
However I did update to the latest published `vue-language-server` package so I also think it’s related to that update. 

I have confirmed that the changes to this project in general has not increased the cpu. 

I’llleave this open for now to track and tag any issues from that project that may be related to this issue.

**Comment by ghost** [comment]
I´m also getting this huge spike, using 240% of my cpu. Is there still some development going on in this repo?
